### ISSUE TITLE
Documentation for writing your own custom loss function

### ISSUE THREAD

**Issue by JamieMair** [PumasAI/SimpleChains.jl#106]
Is there any documentation for implementing a custom loss function, e.g. something similar to `LogitCrossEntropyLoss`? 

I have been looking here - https://github.com/PumasAI/SimpleChains.jl/blob/a735c44c976aa699b6263629b4c21c227504ad08/src/loss.jl and some code that looks similar to what I need to write, such as the chain_valgrad! function, but I am lost on what the variable names represent. Specifically, what are `pu`,`p`, `arg`, `m` etc? Any guidance would be appreciated

**Comment by DoktorMike** [comment]
Shouldn't this issue be closed as there's already a MR that fixes it?
